src/xtopcom/xdata/xslash.h

```diff
@@ -11,6 +11,8 @@
 
 NS_BEG2(top, data)
 
+#define SLASH_DELETE_PROPERTY  "SLASH_DELETE_PROPERTY"
```

Comment:
move this to contact's cpp file. in this PR, no one uses it.

Reply:
this is with the xvm pull https://github.com/telosprotocol/xvm/pull/34/files, slash.h is to put common info